book/prologue/README.md

```diff
@@ -58,7 +58,7 @@ when they do arrive there, like first-class functions in C# or parametric
 polymorphism in Java, it's typically in a limited and awkward form. The only
 languages that completely embody these ideas are
 *statically typed, functional programming languages* like OCaml, F#, Haskell,
-Scala, and Standard ML.[OCaml/benefits of]{.idx}
+Scala, Rust and Standard ML.[OCaml/benefits of]{.idx}
```

Comment:
Do you think it makes sense to call Rust functional?

Reply:
I thought it added an interesting dimension to the languages we do list, since it has many of the features except for automatic garbage collection.  I wasn't sure myself though, so I can be persuaded against as well.